website/docs/tutorials/08-cairo-1-support/05-cheatcodes/stop_warp.md

```diff
@@ -0,0 +1,17 @@
+# `stop_warp`
+
+```cairo
+extern fn stop_warp(target_contract_address: felt252) -> Result::<(), felt252> nopanic;
+```
+
+Stops a warp started by [`start_warp`](./start_warp.md).
+
+- `target_contract_address` address that was previously warped.
+
```

Comment:
Should we mention that calling `stop_warp` on non-existent or not-warped contract does nothing?

Reply:
We don't do that in `stop_prank` docs

Reply:
I wonder if we are testing this behavior, we should start mentioning it as well (in other docs too).